### PR TITLE
Do not insert "..." when committing generics in VB

### PR DESCRIPTION
**Pull request by rchande**

### CODE REVIEW

**Comment by rchande** [comment]
This change extracts the parts of @lorcanmooney 's PR https://github.com/dotnet/roslyn/pull/13430 that deal with committing generics.
Tag @dotnet/roslyn-ide for review
Tag @MattGertz for ask mode approval: this fixes https://github.com/dotnet/roslyn/issues/13161, an annoying regression where committing a completion for a generic symbol in VB inserts extra text

**Comment by MattGertz** [comment]
Approved pending the usual paperwork.  

(N.B. probably would not meet the bar later this week, & we'd instead defer to the next RC candidate.)

File: src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb [hunk 1]
```diff
@@ -2524,6 +2524,7 @@ Anonymous Types:
             End Using
         End Function
 
+<<<<<<< HEAD
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb]: I don't see this ending well.

File: src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb [hunk 2]
```diff
@@ -5620,6 +5620,30 @@ Class C
...
+            Dim expected =
+<code>
+Class G(Of T)
+End Class
+
+Class DG
+    Function Bar() as G(Of
+End Class</code>.Value
+
+            Await VerifyProviderCommitAsync(text, "G(Of …)", expected, Nothing, "")
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb]: Why are we committing ... ?

File: src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb [hunk 3]
```diff
@@ -7582,6 +7606,7 @@ End Namespace
             Await VerifyItemIsAbsentAsync(text, "Item9")
             Await VerifyItemIsAbsentAsync(text, "Rest")
         End Function
+<<<<<<< HEAD
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb]: no! :)

File: src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb (hunk 2 above)
Comment (rchande) [inline src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb]: That's the displaytext of the item to commit. The local `expected` contains the expected text which has no `...`.

**Pull request merged**